graphql-server/src/utils/servers.ts

```diff
@@ -48,17 +53,16 @@ export async function initializeApolloServer(
   console.info("Building, generating, and merging schemas...");
   const { neo4jDbSchema, ogm } = await buildNeo4jDbSchema();
   const customSchema = await buildCustomSchema(ogm);
-  const databricksSchema = await buildDatabricksSchema();
   const mergedSchema = mergeSchemas({
-    schemas: [neo4jDbSchema, databricksSchema, customSchema],
```

Comment:
making sure I understand correctly.. databricks interactions are consolidated to the `patients.ts`?

Reply:
Essentially